### PR TITLE
Make the library a package

### PR DESCRIPTION
**Pull request by bekaertruben**
These changes would make the package installable with `pip install git+https://github.com/nu-radio/cr-pulse-interpolator`.  
Please review the information in the `pyproject.toml` file.

### CODE REVIEW

File: pyproject.toml
Comment (MijnheerD) [inline pyproject.toml:1]: This looks good! It installed without problem on my local installation. Though is there no way to lower the Python 3.8 requirement?

**Review by MijnheerD — CHANGES_REQUESTED** [review]
Before we can merge, I think the demo files need to updated with the correct imports as well, no?

File: pyproject.toml
Comment (bekaertruben) [reply pyproject.toml:1]: I've made the requested changes, including lowering python requirement.
Note though that versions older than 3.8 are end-of-life, and may be insecure

**Review by MijnheerD — APPROVED** [review]
Looks clean, thanks for the work!

**Comment by MijnheerD** [comment]
Oh, so apparently I don't have the rights to review it for merger :-( 

@acorstanje or @anelles , could you please put your review in?

**Comment by anelles** [comment]
@MijnheerD can you now? (working on the settings)

**Comment by MijnheerD** [comment]
@anelles Yes, it works now 💯 My review has granted merge access to the PR. 

@bekaertruben , usually we handle a 24h waiting period between the review and actually merging the changes, so feel free to merge later today!

**Pull request merged**
